mta/tests/migration/controls/test_business_services.py

```diff
@@ -0,0 +1,25 @@
+import pytest
+from pytest_check import check
+
+from conftest import generate_string
+
+
+@pytest.mark.bs
+def test_business_service_crud(business_service, stakeholder, get_api, delete_api, update_api):
+    # get business_service_from_db
+    business_service_from_db = get_api.businessservices_id_get(business_service.id)
+
+    # Assertion checking the created business services's name
+    check.equal(business_service_from_db.name, business_service.name)
+
+    # Edit the BS name and description
+    business_service.name = generate_string(start="update name")
+    business_service.description = generate_string(length=120, start="update description")
```

Comment:
@kpunwatk since you update the description, please add a check for it too or remove this line

Reply:
Done and test pass too https://main-jenkins-csb-migrationqe.apps.ocp-c1.prod.psi.redhat.com/view/MTA/job/mta/job/mta-api-tests-runner/9/console

Reply:
Thanks @mguetta1 for suggestion!